cfme/tests/configure/test_access_control.py

```diff
@@ -2577,7 +2576,11 @@ def test_authorized_users_can_login():
             1. User created
             2. User logged in
     """
-    pass
+    user, _ = user_self_service_role
+
+    # login with user having self service role and uppercase password
+    with user:
+        appliance.server.login(user)
```

Comment:
I really like how neatly this has been implemented, but if the user creation is a part of the test, I think it will be better if you put user creation within the test and not as a part of setup i.e. don't use fixture to create user.

Also it kinda looks like it should be a part of smoke test, thoughts on adding a smoke test marker?

Reply:
@valaparthvi I think it's fine having it in the fixture. I think creating the user is really a part of the setup, and the point of the test is ensuring that the user is able to login. That said, I'd be fine if you want to include the user creation as part of the test, so up to you @dgaikwad.

Reply:
@valaparthvi @john-dupuy I have added user creation inside test and also put a smoke marker.